x-pack/plugins/apm/common/agent_name.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getNormalizedAgentName } from './agent_name';
+
+describe('agent name helpers', () => {
+  describe('getNormalizedAgentName', () => {
+    describe('with an already normalized name', () => {
+      it('returns the original', () => {
+        expect(getNormalizedAgentName('nodejs')).toEqual('nodejs');
+      });
+    });
```

Comment:
Perhaps a test to for some of the unsupported languages. What should this return for instance?

```ts
        expect(getNormalizedAgentName('cpp')).toEqual('?');
```